recipe/conda_build_config.yaml

```diff
@@ -2,7 +2,7 @@
 # see this: https://github.com/phracker/MacOSX-SDKs/search?q=%3CMetal%2FMTLBlitCommandEncoder.h%3E
 # also some symbols only available after 11.0
 # Please consult conda-forge/core before doing this
-MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
-  - "11.0"                 # [osx and x86_64]
+# MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
+#   - "11.0"                 # [osx and x86_64]
```

Comment:
You need to keep this uncommented

Reply:
You also need to add `__osx` line as described in our docs to the run requirements.

Reply:
Okay, so do this regardless if it is an error or just a warning, right?

Reply:
Ugh... maybe this is why I ended up not doing this to begin with... I never keep track:

```
The reported errors are:
- Encountered problems while solving:
-   - nothing provides requested __osx >=11.0
```

The azure pipeline has this osx info:

```
Starting: Initialize job
Agent name: 'Azure Pipelines 2'
Agent machine name: 'Mac-1648091671806'
Current agent version: '2.200.2'
Operating System
Mac OS X
10.15.7
19H1713

```

what do you advise here?

Reply:
Need to change this parameter:

https://github.com/ngam/libtvm-feedstock/blob/989c2a8b3eb223886d5118d6ee58b21bcf47ff64/.azure-pipelines/azure-pipelines-osx.yml#L8

i will try to improvise...